.devcontainer/setup.sh

```diff
@@ -7,8 +7,8 @@ curl -fsSL https://raw.githubusercontent.com/speakeasy-api/speakeasy/main/instal
 rmdir samples || true
 mkdir samples
 
-npm install
-npm install -g ts-node
+npm install --ignore-scripts
```

Comment:
Why this change? Is this from you or the generator?

Given how much of the changes are auto-generated, it would be helpful for you to comment at the individual places where you actually made changes vs where the generator is responsible for them.  That'll focus our attention to the right places. Thanks!